lib/-private/todo-handler.js

```diff
@@ -64,7 +70,11 @@ module.exports = class TodoHandler {
           result.column === todo.column
       );
 
-      result.severity = TODO_SEVERITY;
+      if (todo.warnDate instanceof Date && today > todo.warnDate) {
+        result.severity = WARNING_SEVERITY;
+      } else {
+        result.severity = TODO_SEVERITY;
+      }
```

Comment:
I'd like to add a utility function for this over in the todo utils. 

```js
result.severity = getTodoSeverity(todo);
```

^ would allow both the eslint formatter and ember-template-lint to use the same shared func and allow you to remove the additional early return just above this area (because `getTodoSeverity(todo)` would return `ERROR_SEVERITY` when the `todo.errorDate` has already passed)

Reply:
And you'd evaluate the date from the passed in todo, and pass back the right severity? Seems like a good idea.

Reply:
Also, seems severe... :)

Reply:
> And you'd evaluate the date from the passed in todo, and pass back the right severity?

Yeah exactly. This will help us ensure that our two clients stay in sync over time (especially worried WRT the timezone thing, and encapsulating that into the utils repo helps centralize the logic).

Reply:
Confirm.

Reply:
Done.